winterfell/controllers/authController.js

```diff
@@ -1,5 +1,6 @@
 var _ = require('lodash');
 var passport = require('passport');
+var requestify = require('requestify');
```

Comment:
Do we need a new library just to do this POST request to ID.me? Would prefer to minimize the number of external libs we have, it will make the project a lot easier to maintain